src/stacktrace_windows.cpp

```diff
@@ -120,7 +120,7 @@ namespace {
       std::string lineInformation;
       std::string callInformation;
       if (SymFromAddr(GetCurrentProcess(), addr, &displacement64, symbol)) {
-         callInformation.append(" ").append({std::string(symbol->Name), symbol->NameLen});
```

Comment:
The reason append was used before and not the `std::` constructor was that I was not 100% sure the symbol would not contain null characters. The append function which is almost identical to the constructor for these arguments declare that null characters are fine to add. 

Are you sure that the `symbol` string will never have null characters? Is this documented?
https://en.cppreference.com/w/cpp/string/basic_string/append

Reply:
Ignore that. I see now that the declaration for `std::string constructor (4)` is identical. 
https://en.cppreference.com/w/cpp/string/basic_string/basic_string

So this change is slightly better as it creates fewer copies (potentially)